next.config.js

```diff
@@ -15,23 +13,23 @@ module.exports = withSourceMaps(withCSS({
         'process.env.SENTRY_DSN': JSON.stringify(process.env.SENTRY_DSN || 'https://49af7fff247c445b9a7c98ee21ddfd2f@sentry.io/1427510'),
       })
     )
-    config.module.rules.push({
-      test: /\.(otf|ttf|woff|woff2)$/,
-      use: {
-        loader: 'file-loader',
-        options: {
-          limit: 10000,
-          publicPath: '/_next/static/',
-          outputPath: 'static/',
-          name: '[name].[ext]'
-        }
-      }
-    })
+    // config.module.rules.push({
```

Comment:
Why is this commented out? Maybe a comment should be added to the top explaining why it's being commented out and when it should be uncommented back?

Reply:
I was checking if that configuration had any effect at all. I removed it now because it doesn't.